### ISSUE TITLE
list service endpoints returns configurations and not actual list of SEPs

### ISSUE THREAD

**Issue by nanjum88** [Appdynamics/api-commandline-tool#32]
Trying to get the list of all the SEPs in an application , but the list command returns SEP configuration, was expecting all the SEPs detected instead. 

`act.sh -E saas-prod sep list -a 2973 | jq .
{
  "applicationId": "2973",
  "applicationName": "AIQ-HomePoint",
  "sepContainer": [
    {
      "entityType": "APPLICATION",
      "entityId": "2973",
      "agentType": "DOT_NET_APP_AGENT",
      "override": true
    },
    {
      "entityType": "APPLICATION",
      "entityId": "2973",
      "agentType": "NODEJS_APP_AGENT",
      "override": true
    },
    {
      "entityType": "APPLICATION",
      "entityId": "2973",
      "agentType": "PHP_APP_AGENT",
      "override": true
    },
    {
      "entityType": "APPLICATION",
      "entityId": "2973",
      "agentType": "APP_AGENT",
      "override": true,
      "sEPMatchPointConfigs": [
        {
          "id": "20817",
          "name": "EJB",
          "enabled": false,
          "entryPointType": "EJB",
          "namingSchemeType": "EJB_NAME_AND_METHOD_NAME"
        },
        {
          "id": "20813",
          "name": "JMS",
          "enabled": true,
          "entryPointType": "JMS",
          "namingSchemeType": "DESTINATION_NAME"
        },
        {
          "id": "20814",
          "name": "POJO",
          "enabled": true,
          "entryPointType": "POJO",
          "namingSchemeType": "FULLY_QUALIFIED_CLASS_NAME"
        },
        {
          "id": "20816",
          "name": "SERVLET",
          "enabled": false,
          "entryPointType": "SERVLET",
          "namingSchemeType": "URI",
          "namingSchemeProperties": "uri-length;first-n-segments;segment-length;2;"
        },
        {
          "id": "20812",
          "name": "SPRING_BEAN",
          "enabled": false,
          "entryPointType": "SPRING_BEAN",
          "namingSchemeType": "BEAN_ID_AND_METHOD_NAME"
        },
        {
          "id": "20815",
          "name": "STRUTS_ACTION",
          "enabled": true,
          "entryPointType": "STRUTS_ACTION",
          "namingSchemeType": "ACTION_NAME_AND_METHOD_NAME"
        },
        {
          "id": "20818",
          "name": "WEB_SERVICE",
          "enabled": true,
          "entryPointType": "WEB_SERVICE",
          "namingSchemeType": "SERVICE_NAME_AND_OPERATION_NAME"
        }
      ]
    }
  ],
  "actions": [
    {
      "href": "http://elliemae-prod.saas.appdynamics.com/controller/api/accounts/56/applications/2973/sep",
      "method": [
        "POST"
      ],
      "name": "SEP_CREATE_UPDATE"
    }
  ],
  "links": [
    {
      "href": "http://elliemae-prod.saas.appdynamics.com/controller/api/accounts/56/applications/2973/sep/{tier.id}",
      "name": "sep"
    }
  ]
}
`
@svrnm

**Comment by svrnm** [comment]
Good point. That should probably be named differently. I'll check that.

**Comment by nanjum88** [comment]
Thanks @svrnm . 
Is there a way to get the actual list of SEPs from the controller and not the SEP configurations ?

**Comment by svrnm** [comment]
fixed with the latest release, please try it out and let me know if it works:-)
